packages/lib/bookings/filterHostsByLeadThreshold.test.ts

```diff
@@ -27,18 +26,10 @@ afterEach(() => {
 });
 
 describe("filterHostByLeadThreshold", () => {
-  it("skips filter if host is fixed", async () => {
-    const hosts = [{ isFixed: true, createdAt: new Date(), user: { id: 1, email: "example1@acme.com" } }];
-    expect(
-      filterHostsByLeadThreshold({
-        hosts,
-        maxLeadThreshold: 3,
-        eventTypeId: 1,
-      })
-    ).resolves.toStrictEqual(hosts);
-  });
   it("skips filter if lead threshold is null", async () => {
-    const hosts = [{ isFixed: false, createdAt: new Date(), user: { id: 1 } }];
+    const hosts = [
+      { isFixed: false as const, createdAt: new Date(), user: { id: 1, email: "member1-acme@example.com" } },
```

Comment:
We need to mark isFixed as const so it's not type hinted `boolean` - passing a boolean type errors.